pip-dep/requirements.txt

```diff
@@ -5,7 +5,7 @@ lz4>=2.1.6
 msgpack>=0.6.1
 numpy>=1.16.0
 tblib>=1.4.0
-torch==1.3
+torch>=1.3
```

Comment:
Perhaps, use `torch==1.3.*` or `torch~=1.3.0` (which is probably same) to use latest torch version only from 1.3 stream.
However, I'd check pytorch updates policy to ensure they don't break things in patch-releases :)